docs/users/extend/in-container-configuration.md

```diff
@@ -1,11 +1,12 @@
 ## In-Container home directory configuration
 
-There are a number of reasons that your web container home directory may need custom configuration. You may want your normal git configuration in there, or a composer auth.json, etc.
+Custom shell config (bash or your preferred shell), your usual git configuration, a composer auth.json and more can be achieved within your containers.  Place all your dotfiles in your `~/.ddev/homeadditions` directory and DDev will use these in all your projects' Docker containers.
 
 On `ddev start`, ddev attempts to create a user inside the web and db containers with the same name and use id as the one you have on the host.
 
-If you have a ~/.ddev/homeadditions
-If you have a ~/.ddev/homeadditions directory (in the global .ddev directory) or your project has a .ddev/homeadditions directory, their contents will be copied recursively into the in-container home directory during `ddev start`. (Note that project homeadditions contents override the global homeadditions.)
+DDev looks for `homeadditions` directory either in `~/.ddev/homeadditions` (the global .ddev directory) or the `.ddev/homeadditions` directory of a particular project, and will copy their contents recursively into the in-container home directory during `ddev start`. (Note that project homeadditions contents override the global homeadditions.)
+
+If you want to use a shell other than bash, add it as a package to [customize the Docker image](https://ddev.readthedocs.io/en/stable/users/extend/customizing-images/).
```

Comment:
I don't see a reason for this line to be here, as it's on a completely different topic. But `webimage_extra_packages` would be the way to do it, that's covered over there.

Reply:
The benefit I see in that line is if someone's thinking "I want to customize my shell" and they go into that page but that page would then be insufficient if their desired shell customizations require a different shell, so to save them from having to then read through the menu/index and try to find the right page (which might be easy enough for some people) linking to that other page directly could make it easier for others.